library/pluginsystem/PluginSystem.php

```diff
@@ -76,7 +76,7 @@ public function init()
         $composerJsonPath = realpath( __DIR__."/../../composer.json" );
         $composerJson = file_get_contents( $composerJsonPath );
         if ( $composerJson === false ) {
-            throw new Exception( "$composerJsonPath/composer.json is not readable" );
+            throw new \Exception( "$composerJsonPath/composer.json is not readable" );
```

Comment:
EDIT:  I thought this was a goofup, until I realized that PHP must have stupidly decided to use a single backslash as the operator for a namespace....